armory/scenarios/multimodal_so2sat_scenario.py

```diff
@@ -12,7 +12,7 @@
 
 
 class So2SatClassification(Scenario):
-    def __init__(self, *args, **kwargs):
+    def __init__(self, config, *args, **kwargs):
```

Comment:
Given that the scenario is always instantiated here with `config` and `kwargs`, I thought having both `config` and `*args` was a little confusing since I don't think there will ever be any args passed besides `config`

Reply:
The base scenario doesn't have a `*` delimiter, so you can technically pass in the kwargs arguments as args.

Reply:
Perhaps a modification to add that?